CefSharp.Core/Internals/StreamAdapter.h

```diff
@@ -17,6 +17,7 @@ namespace CefSharp
         {
             CriticalSection _syncRoot;
             gcroot<Stream^> _stream;
+            bool isMemoryStream = false;
```

Comment:
In `C++` we prefix with `_`

(I would like to make the `C#` code consistent, though I've had some resistance).